### PR TITLE
update project ui

### PR DESCRIPTION
**Pull request by Aliaksandrzaz**

### CODE REVIEW

File: js/src/features/project/components/ProjectHeaderMenu.tsx [hunk 1]
```diff
@@ -1,37 +1,72 @@
...
+
+import "../project.scss";
+
+const { Paragraph } = Typography;
 
 type Props = {
   slug: string;
 };
 
 export const ProjectHeaderMenu: React.FC<Props> = ({ slug }) => {
```
Comment (kwoon) [inline js/src/features/project/components/ProjectHeaderMenu.tsx:18]: Looks like it is dumb component. Let try to send `project` via props.
Before send make sure that project is not empty (I hate endless `project?.` view)

Comment (kwoon) [reply js/src/features/project/components/ProjectHeaderMenu.tsx:18]: And you don't need `slug` prop coz it's already exists in `project`

File: js/src/features/project/components/ProjectHeaderMenu.tsx [hunk 2]
```diff
@@ -1,37 +1,72 @@
...
+          selectedKeys={[location.pathname]}
+          className="page-header__menu"
+        >
+          <Menu.Item key={`/projects/${slug}/role`}>
+            <Link to={`/projects/${slug}/role`}>
+              <span>Role</span>
+            </Link>
+          </Menu.Item>
+          <Menu.Item
+            key={location.pathname.includes("teams") ? location.pathname : ""}
```
Comment (kwoon) [inline js/src/features/project/components/ProjectHeaderMenu.tsx:39]: Why do you check only `teams` url ?

File: js/src/features/project/components/ProjectHeaderMenu.tsx [hunk 3]
```diff
@@ -1,37 +1,72 @@
...
+    </PageHeader>
   );
 };
+
+type StatusProps = {
+  status?: ProjectStatusTypeResponse;
+};
+
+const Status: React.FC<StatusProps> = ({ status }) => {
+  if (!status) return null;
```
Comment (kwoon) [inline js/src/features/project/components/ProjectHeaderMenu.tsx:69]: Useless check. Status can't be null. That means is useless too. You can remove it

File: js/src/features/project/components/Project/ProjectStatus.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            Activate
+          </Button>
+        </div>
+      )}
+
+      {project.status !== "Suspended" && (
+        <div className="status__item">
+          <span> Pause the project. You can return to work later.</span>
+          <Button
+            value="Suspended"
```
[the remark this answers is not in the document]
Comment (Aliaksandrzaz) [reply js/src/features/project/components/Project/ProjectStatus.tsx:33]: It is not necessary. Fixed.

File: js/src/features/project/components/ProjectHeaderMenu.tsx (hunk 3 above)
Comment (Aliaksandrzaz) [reply js/src/features/project/components/ProjectHeaderMenu.tsx:69]: Fixed.

File: js/src/features/project/components/ProjectHeaderMenu.tsx (hunk 1 above)
Comment (Aliaksandrzaz) [reply js/src/features/project/components/ProjectHeaderMenu.tsx:18]: Done

File: js/src/features/project/components/ProjectHeaderMenu.tsx (hunk 2 above)
Comment (Aliaksandrzaz) [reply js/src/features/project/components/ProjectHeaderMenu.tsx:39]: If I am on the page of a specific project, then I need the tab to be highlighted "Teams". You cannot put an array in `key` for verification, so I did such a verification.

Comment (kwoon) [reply js/src/features/project/components/ProjectHeaderMenu.tsx:39]: But what about roles or settings?

Comment (Aliaksandrzaz) [reply js/src/features/project/components/ProjectHeaderMenu.tsx:36]: I checking roles and setting
`key={'/projects/${project.slug}/role'}`
`key={'/projects/${project.slug}/settings'}`

**Pull request merged**